hostinit/setup.go

```diff
@@ -707,13 +707,10 @@ func (init *HostInit) fetchRemoteTaskData(ctx context.Context, taskId, cliPath,
 	}
 	sshOptions = append(sshOptions, "-o", "UserKnownHostsFile=/dev/null")
 
-	// When testing, use this writer to force a copy of the output to be written to standard out so
-	// that remote command failures also show up in server log output.
-	//cmdOutput := io.MultiWriter(&util.CappedWriter{&bytes.Buffer{}, 1024 * 1024}, os.Stdout)
-
 	cmdOutput := &util.CappedWriter{&bytes.Buffer{}, 1024 * 1024}
 	makeShellCmd := &subprocess.RemoteCommand{
-		CmdString:      fmt.Sprintf("%s -c '%s' fetch -t %s --source --artifacts --dir='%s'", cliPath, confPath, taskId, target.Distro.WorkDir),
+		Id:             fmt.Sprintf("fetch-artifacts-%s", taskId),
+		CmdString:      fmt.Sprintf("%s -c %s fetch -t %s --source --artifacts --dir='%s'", cliPath, confPath, taskId, target.Distro.WorkDir),
```

Comment:
we should probably log this sometime? I'm not sure that Run does, but it might, if it doesn't, it'd be reasonable to add a log message

Reply:
RemoteCommand logs this already.